apps/admin-gui/src/app/shared/components/two-entity-attribute-page/two-entity-attribute-page.component.ts

```diff
@@ -86,15 +96,15 @@ export class TwoEntityAttributePageComponent implements OnInit {
         switch (this.secondEntity){
           case 'resource':
             this.resourcesManagerService.getAssignedResourcesWithGroup(this.firstEntityId).subscribe(resources => {
-              this.entityValues = resources;
+              this.entityValues = resources.sort(this.compareFnName);
               this.preselectEntity();
               this.loading = false;
             });
             break;
           case 'member':
             // return one attribute because if an empty list is passed, all attributes are returned
             this.membersManager.getCompleteRichMembersForGroup(this.firstEntityId, false, this.allowedStatuses, null, [Urns.MEMBER_CORE_ID]).subscribe(members => {
-              this.entityValues = members;
+              this.entityValues = members.sort(this.compareFnUser);
```

Comment:
when we are sorting the names(first name + last name) of members and users I think we really want to sort according to last name(then according to firstname) + count with czech alphabet